src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/ArtistWrapper.tsx

```diff
@@ -108,7 +113,14 @@ export class VanguardArtistWrapper extends React.Component<
         >
           <BackgroundContainer>{background}</BackgroundContainer>
           <ArticleWithFullScreen article={article}>
-            <ArtistContainer pb={4} maxWidth={["100vw", 1000]} px={4} mx="auto">
+            <ArtistContainer
+              pb={4}
+              maxWidth={["100vw", 1000]}
+              px={4}
+              mx="auto"
+              isMobile
```

Comment:
I would expect this to use a prop, it looks like we are setting the `ArtistContainer` `isMobile` to `true` for all instances.

Reply:
Ah, ok yeah at first glance it does look like its always `true` but `isMobile` is a boolean prop passed here from the `SeriesWrapper` component:

```
{relatedArticles &&
        relatedArticles.map((artistArticle, i) => (
          <VanguardArtistWrapper
            key={i}
            article={artistArticle}
            section={slugifiedTitle}
            isMobile={isMobile}
          />
        ))}
```

Reply:
I would still expect this to use `isMobile={isMobile}` if we are passing a value, the declaration in the code should mean always true.